src/test/java/org/apache/cassandra/sidecar/server/ServerTest.java

```diff
@@ -181,6 +194,25 @@ void updatingTrafficShapingOptions(VertxTestContext context)
               });
     }
 
+    @Test
+    @DisplayName("Updating traffic shaping options with non-zero listen port should succeed")
+    void updateTrafficShapingOptionsWithNonZeroListenPort()
+    {
+        configureServer("config/sidecar_single_instance_default_port.yaml");
+
+        try
+        {
+            server.start().toCompletionStage().toCompletableFuture().get(30, TimeUnit.SECONDS);
+            TrafficShapingOptions update = new TrafficShapingOptions()
+                                           .setOutboundGlobalBandwidth(100 * 1024 * 1024);
+            server.updateTrafficShapingOptions(update);
+        }
+        catch (Exception exception)
+        {
+            Assertions.fail(exception);
+        }
```

Comment:
I guess the catch is not necessary. Is it added because IDE complains no assertion is performed? 
If so, how about this?
```java
        assertThatNoException().isThrownBy(() -> {
            server.start().toCompletionStage().toCompletableFuture().get(30, TimeUnit.SECONDS);
            TrafficShapingOptions update = new TrafficShapingOptions()
                                           .setOutboundGlobalBandwidth(100 * 1024 * 1024);
            server.updateTrafficShapingOptions(update);
        });
```

Reply:
Without the fix, the exception was thrown and I would fail the test. I will update the test. I was running `@RepeatedTest` so it was necessary to fail the test so the server was properly closed during teardown